### DISCUSSION TITLE
How to access `author` meta data with meta shortcode

### DISCUSSION THREAD

**Discussion by kompre** [quarto-dev/quarto-cli#7047] [Q&A]
### Description

I would like to access `author` metadata within a document using the shortcode `{{< meta author.<key> >}}`, but it seems I can only use `{{< meta author >}}` that wil return the name of the author.

Here a minimal working example:

````{qmd}
---
title: "author_meta"
author:
    - name: pippo
      number: 12345679
      email: abc@xyz.it
---

## Quarto

{{< meta author >}}

{{< meta author.name >}}

{{< meta author.number >}}

{{< meta author.email >}}
````

It render as: 

![image](https://github.com/quarto-dev/quarto-cli/assets/11796345/b5b42b92-ecd1-4a25-86b2-b5cb4d7203d9)

I've also have tried other configuration such as:

```{yaml}
---
title: "author_meta"
author:
  name: pippo
  number: 12345679
  email: abc@xyz.it
---
```

But still only the name value is rendered with the `{{< meta author >}}` shortcode.

How do I access the rest of the information?

**Comment by mcanouil** [comment]
Currently you can't, see:
- https://github.com/quarto-dev/quarto-cli/issues/6769
